components/apps/ptp-operator/base/subscription.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ptp-operator-subscription
+  namespace: openshift-ptp
+spec:
+  channel: patch-me
+  name: ptp-operator
+  source: redhat-operators
```

Comment:
This looks good for now, but consider disconnected use cases where we mirror the catalog and need to adjust the catalogsource.

Reply:
Noted.